platform/binance/client.go

```diff
@@ -88,6 +89,26 @@ func (c *Client) GetTxsOfAddress(address string, token string) (*TxPage, error)
 	return stx, nil
 }
 
+func (c *Client) GetTransactionReceipt(hash string) (*Receipt, error) {
+	url := fmt.Sprintf("%s/tx/%s?format=json", c.RPCBaseURL, hash)
+	println(url)
+	res, err := c.HTTPClient.Get(url)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
+
+	recp := new(Receipt)
+	err = json.NewDecoder(res.Body).Decode(recp)
+	if err != nil {
```

Comment:
I think you can just return err here and recp, you don't need to have another handle error